### DISCUSSION TITLE
Free S3-compatible storage for Pocketbase (10GB free) with IDrive

### DISCUSSION THREAD

**Discussion by burggraf** [pocketbase/pocketbase#3822] [Show and tell]
I just tried IDrive and it works with Pocketbase as free S3 storage.  You get 10GB free:
https://app.idrivee2.com/signin

I haven't uploaded any files yet, but the backups to S3 works fine, and I can set up an S3 storage provider just fine for uploading files, so that should work too.

**Comment by burggraf** [comment]
I got an email from iDrive that they are discontinuing their 10GB free plan, but I confirmed with the company that it does not apply to the S3-compatible storage option, so the 10GB free S3 storage will continue.  They're also offering $15/yr for the first year for 1TB (1000GB) of storage and a regular price of $4/mo per TB.  Pretty cheap.

**Reply by cheskoxd** [reply]
Thanks for the info, I've never used iDrive, I've worked with AWS3 and Backblaze

Backblaze has a 10GB free tier and a pay as you go plan.

It's also really cheap, it's $6 x TB

https://www.backblaze.com/cloud-storage/pricing

In case you want another option for some storage for testing or a small app, you could use their free tier and works great (it's also really scalable)

**Reply by tigawanna** [reply]
> I got an email from iDrive that they are discontinuing their 10GB free plan, but I confirmed with the company that it does not apply to the S3-compatible storage option, so the 10GB free S3 storage will continue.  They're also offering $15/yr for the first year for 1TB (1000GB) of storage and a regular price of $4/mo per TB.  Pretty cheap.
> 
Interesting, thanks for the link

**Reply by imzza** [reply]
iDrive advertises 10 GB of monthly egress, but when I contacted support after reaching the limit, I was told that it is actually a one-time allowance. Once you use the full 10 GB, you are locked out and cannot download your own files unless you upgrade your plan.

This is extremely disappointing and one of the poorest customer experiences I have encountered. Customers should be clearly informed about this limitation before signing up.
